domain/user/state/state.go

```diff
@@ -291,6 +291,86 @@ func (st *State) SetPasswordHash(ctx context.Context, uuid user.UUID, passwordHa
 	})
 }
 
+// EnableUser will enable the user with the supplied UUID. If the user does not
+// exist an error that satisfies usererrors.NotFound will be returned. If the
+// user is already enabled an error that satisfies usererrors.AlreadyEnabled
```

Comment:
Update this comment. Error doesn't exist ?

Reply:
Don't think we need to worry about this error for now. We can just treat it as a no op

Reply:
Yep, that was my first implementation, this comment is a rudiment. Fixed